charts/redpanda/chart.go

```diff
@@ -90,38 +77,46 @@ func Template(release helmette.Release, values PartialValues, kubeConfig kube.Co
 		PostInstallUpgradeJob(dot),
 	}
 
-	manifests = append(manifests, asObj(ConfigMaps(dot))...)
-	manifests = append(manifests, asObj(CertIssuers(dot))...)
-	manifests = append(manifests, asObj(RootCAs(dot))...)
-	manifests = append(manifests, asObj(ClientCerts(dot))...)
-	manifests = append(manifests, asObj(ClusterRoleBindings(dot))...)
-	manifests = append(manifests, asObj(ClusterRoles(dot))...)
-	manifests = append(manifests, asObj(LoadBalancerServices(dot))...)
-	manifests = append(manifests, asObj(Secrets(dot))...)
-
-	j := 0
-	for i := range manifests {
-		// Nil unboxing issue
-		if reflect.ValueOf(manifests[i]).IsNil() {
-			continue
-		}
-		manifests[j] = manifests[i]
-		j++
+	// NB: gotohelm doesn't currently have a way to handle casting from
+	// []Instance -> []Interface as doing so generally requires some go
+	// helpers.
+	// Instead, it's easiest (though painful to read and write) to iterate over
+	// all functions that return slices and append them one at a time.
```

Comment:
I fell like `asObj` would work in gotohelm transpiler. I liked how previous neat append list was constructed.

There is only one change to the `asObj` function as nitialization of array is not supported in our transpiler.
```go
func asObj[T kube.Object](manifests []T) []kube.Object {
	out := []kube.Object{}
	for i := range manifests {
		out = append(out, manifests[i])
	}
	return out
}
```

Reply:
Oh, right! Feel free to resurrect that if you'd like.